### ISSUE TITLE
nNA not showing all NAs

### ISSUE THREAD

**Issue by abeburnett** [data-cleaning/validate#75]
Hi there,

I'm trying to detect the incidence of NAs in a data frame, overall, and by field. I see that there's an nNA column in the summary() of the confronted data frame. But, it only seems to detect NAs where the entire column was previously NA. 

For instance, if I start with a data frame which has everything populated, change one value in a column to NA, and then run the validator/confront, I see that it doesn't detect any NAs. 

If, however, I start with a column comprised of all NAs and then populate one of the cells with a legitimate value, then the NA count is displayed, and accurate. 

It seems like the desirable behavior is that nNA should always show the sum of NAs in whatever column, regardless of what validation expression is being run on the column.

Can you please comment on this? Thank you!

**Comment by markvanderloo** [comment]
The number of `NA`'s counted by nNA is not the number of NA's in a record. It is the number of 'validation events' that result in an NA. For example, if you have a rule that states 
```r
mean(x) > 0
```
This will result in a single `NA`, if `x` contains at least one missing value. However, the rule

```r
x > 0
```
results in `length(x)` logical values, and each missing value is counted.

Not sure if this clears things up for you. A small, explicit example would help.

-M

**Comment by abeburnett** [comment]
Ah, that's very helpful. I get it now. Thanks for the clarity!
